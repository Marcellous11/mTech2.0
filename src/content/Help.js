import React from 'react';
import '../styles/Help.css';
const Help = () => {
	return (
		<div className="Help">
			<div className="Help-Options">
				<ul className="Help-Option-Ul">
					<li>
						{' '}
						<b>help</b> ...............Show all commands
					</li>
					<li>
						{' '}
						<b>about me</b>..................A short bio
					</li>
					<li>
						<b>skills</b>...................My strengths
					</li>
					<li>
						<b>work</b>.......Some of my work experience
					</li>
					<li>
						<b>projects</b>.........My favorite projects
					</li>
					<li>
						<b>clear </b>...........Clear terminal window
					</li>
					<li style={{ color: 'rgb(0,214,0)' }}>
						{' '}
						<b>all</b> ..................Show everything
					</li>

					<li style={{ color: 'red' }}>
						<b style={{ color: 'red' }}>x </b>......................Leave Terminal
					</li>
				</ul>
				<ul className="Help-Option-Ul-Mini">
					<li>
						{' '}
						<b>help</b> <p> - Show all commands</p>
					</li>
					<li>
						{' '}
						<b>about me</b> <p> - A short bio</p>
					</li>
					<li>
						<b>skills</b>
						<p> - My strengths</p>
					</li>
					<li>
						<b>work</b> <p> - Some of my work experience</p>
					</li>
					<li>
						<b>clear </b>
						<p> - Clear terminal window</p>
					</li>
					<li>
						<b>projects</b>
						<p> - My favorite projects</p>
					</li>
					<li style={{ color: 'rgb(0,214,0)' }}>
						<b>all</b> <p> - Show everything</p>
					</li>

					<li style={{ color: 'red' }}>
						<b>x </b> <p> - Leave Terminal</p>
					</li>
				</ul>
			</div>
		</div>
	);
};
export default Help;
