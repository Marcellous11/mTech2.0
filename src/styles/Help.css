.Help-Option-Ul li {
	margin-left: 2rem;
	color: rgb(186, 184, 49);
	/* font-size: 20px; */
	font-weight: 600;
	list-style-type: none;
	line-height: 2em;
}
.Help-Option-Ul b {
	color: white;
}

.Help-Option-Ul-Mini {
	display: none;
}
@media only screen and (max-width: 830px) {
	.Help-Option-Ul li {
		margin-left: -3em;
		display: none;
	}

	.Help-Option-Ul-Mini {
		display: block;
		margin-left: -2.5em;
		color: white;
		list-style: none;
	}
	.Help-Option-Ul-Mini li p {
		margin: .5em;
		/* font-weight: 700; */
	}
}
