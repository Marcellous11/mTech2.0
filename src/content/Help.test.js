import { logRoles, prettyDOM, render, screen } from '@testing-library/react';
import Help from './Help';

test('Smoke test, does the component render', () => {
	render(<Help />);
});

test('Snapshot ', () => {
	const { asFragment } = render(<Help />);

	expect(asFragment()).toMatchSnapshot();
});

test('should have About me', () => {
	const { getByText, getAllByText, findByText } = render(<Help />);

	expect(getAllByText(`about me`)).toBeInstanceOf(Array);
	expect(getAllByText(`about me`)[0]).toBeInTheDocument();
});
