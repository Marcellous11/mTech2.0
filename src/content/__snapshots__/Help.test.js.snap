// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot  1`] = `
<DocumentFragment>
  <div
    class="Help"
  >
    <div
      class="Help-Options"
    >
      <ul
        class="Help-Option-Ul"
      >
        <li>
           
          <b>
            help
          </b>
           ...............Show all commands
        </li>
        <li>
           
          <b>
            about me
          </b>
          ..................A short bio
        </li>
        <li>
          <b>
            skills
          </b>
          ...................My strengths
        </li>
        <li>
          <b>
            work
          </b>
          .......Some of my work experience
        </li>
        <li>
          <b>
            projects
          </b>
          .........My favorite porjects
        </li>
        <li>
          <b>
            clear 
          </b>
          ...........Clear terminal window
        </li>
        <li
          style="color: rgb(0, 214, 0);"
        >
           
          <b>
            all
          </b>
           ..................Show everything
        </li>
        <li
          style="color: red;"
        >
          <b
            style="color: red;"
          >
            x 
          </b>
          ......................Leave Terminal
        </li>
      </ul>
      <ul
        class="Help-Option-Ul-Mini"
      >
        <li>
           
          <b>
            help
          </b>
           
          <p>
             - Show all commands
          </p>
        </li>
        <li>
           
          <b>
            about me
          </b>
           
          <p>
             - A short bio
          </p>
        </li>
        <li>
          <b>
            skills
          </b>
          <p>
             - My strengths
          </p>
        </li>
        <li>
          <b>
            work
          </b>
           
          <p>
             - Some of my work experience
          </p>
        </li>
        <li>
          <b>
            clear 
          </b>
          <p>
             - Clear terminal window
          </p>
        </li>
        <li>
          <b>
            projects
          </b>
          <p>
             - My favorite porjects
          </p>
        </li>
        <li
          style="color: rgb(0, 214, 0);"
        >
          <b>
            all
          </b>
           
          <p>
             - Show everything
          </p>
        </li>
        <li
          style="color: red;"
        >
          <b>
            x 
          </b>
           
          <p>
             - Leave Terminal
          </p>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
